apps/vaporgui/VHBoxWidget.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include <QFrame>
+#include <QHBoxLayout>
+
+#include <string>
+
+#include "VLineItem.h"
+
+//! \class VContainer
+//!
+//! A simple wrapper class for one or more VWidgets.
+//! Sets a standard layout and margin policy, and protects
+//! against unwanted mouse scroll whell events.
+
+class VHBoxWidget : public QFrame {
+    Q_OBJECT
+
+public:
+//    virtual QSize sizeHint() const;
```

Comment:
It broke the GUI layout so I had to remove it.

Reply:
Hmm.  I tried re-adding it and it seemed to work to me.  Do you remember how it broke?  I'd like to keep this since I was asked to add it, but if it really is broken I can revisit it.